src/ProgressOnderwijsUtils/Collections/MaybeExtensions.cs

```diff
@@ -290,8 +282,7 @@ public static void IfError<TOk>(this Maybe<TOk, Unit> state, Action ifError)
         public static IEnumerable<TError> WhereError<TOk, TError>(this IEnumerable<Maybe<TOk, TError>> maybes)
         {
             foreach (var state in maybes) {
-                var isOk = state.TryGet(out _, out var error);
-                if (!isOk) {

```

Comment:
Enig idee of dit een bug is? Klinkt wel problematisch, dit soort beperkingen; wordt extract variable ineens tricky.

Reply:
Ik zal eens kijken.

Reply:
Lijkt een bug te zijn die zich alleen voor doet met 5.0.

Reply:
Ik maak wel een issue.

Reply:
(als het je maar niet te veel tijd kost; iets met rabbit holes enzo)

Reply:
(Geen bezwaar tegen deze workaround in de tussentijd, trouwens)

Reply:
Ik doe het wel. Er zijn nogal veel issues met CS8602, dus het is vast een duplicaat, maar ach.

Reply:
Bij nader inzien was die warning er altijd al.